### PR TITLE
Code samples 

### PR DESCRIPTION
**Pull request by ihsandemir**

### CODE REVIEW

**Comment by ihsandemir** [comment]
verify

File: examples/serialization/portable/main.cpp [hunk 1]
```diff
@@ -54,7 +54,7 @@ class Person : public hazelcast::client::serialization::Portable {
     std::auto_ptr<std::string> name;
 };
 
-namespace std{
+namespace std {
```
Comment (sancar) [inline examples/serialization/portable/main.cpp]: This usage does not seem right. You don't need to put your `operator<<` in `namespace std`  . There is ADL (Argument Dependent Lookup) just to be able to do this. see
https://en.wikipedia.org/wiki/Argument-dependent_name_lookup

Comment (ihsandemir) [inline examples/serialization/portable/main.cpp]: OK, changed.

**Comment by sancar** [comment]
:+1:

**Pull request merged**